http/peer-id-auth.md

```diff
@@ -0,0 +1,131 @@
+# Peer ID Authentication over HTTP
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2023-01-23  |
+
+Authors: [@MarcoPolo]
+
+[@MarcoPolo]: https://github.com/MarcoPolo
+
+Interest Group: Same as [HTTP](README.md)
+
+## Introduction
+
+This spec defines one way of authenticating Peer IDs over HTTP using a
+challenge-response scheme. The authentication scheme is called `Libp2p-PeerID``
```

Comment:
```suggestion
This spec defines an authentication scheme in accordance with [RFC7235](https://datatracker.ietf.org/doc/html/rfc7235).
The authentication scheme is called `Libp2p-PeerID`.
```

Reply:
https://datatracker.ietf.org/doc/html/rfc9110 is the latest standard :)